mixing/mixclient/client.go

```diff
@@ -159,7 +163,7 @@ func (d *deadlines) restart() {
 type peer struct {
 	ctx    context.Context
 	client *Client
-	rand   io.Reader // non-PRNG cryptographic rand
+	jitter time.Duration
```

Comment:
Could you add a comment here about how a peer's jitter represents how long we delay broadcasts to them (to be recalculated for each new mixing session)? It's a little unintuitive to me otherwise.

Reply:
that's not what this is at all. the jitter value only affects local peers, not remote ones (we can't introduce jitter sending messages to specific peers at all -- each messaging stage is a network broadcast).

it's also not recalculated for each session. it's set once as the peer is created.

Reply:
I'm not super familiar with the `mixing/mixclient` package yet, but after doing some reading (and searching the code for `local peer` and `localpeer`), I'm still struggling to understand what a local peer is, as I can't figure out where it's defined.

I was under the impression that a localpeer was the identity of a remote peer that sent a PR and wants to be included in a session. Given this code: `localPeers map[identity]*peer`. But the comment also says that a local peer is the entity that performs mix sessions, which doesn't quite make sense to me.

Reply:
local peer is one which we created the identity for.

remote peers are other wallets.

the same structure is used to represent both. p.remote is true for remote peers, false for local ones.

Reply:
Got it, thanks. Still, I think a comment about the purpose of per-identity (per-local-peer) jitter would be helpful. The per-message jitter makes sense to me, but I'm struggling to understand why we delay for each identity in addition to delaying for each message.

Reply:
it's to obfuscate any additional delay that a wallet may have broadcasting messages to the rest of the network, e.g. their local clock is not properly synced, or they have added latency due to using tor.

if we change to basing the epoch on network time instead of wall time, we're also down to a second of granularity and this would also help hide that difference.